python/dgl/nn/pytorch/gt/path_encoder.py

```diff
@@ -31,13 +28,19 @@ class PathEncoder(nn.Module):
     >>> import torch as th
     >>> import dgl
     >>> from dgl.nn import PathEncoder
+    >>> from dgl import shortest_dist
 
-    >>> u = th.tensor([0, 0, 0, 1, 1, 2, 3, 3])
-    >>> v = th.tensor([1, 2, 3, 0, 3, 0, 0, 1])
-    >>> g = dgl.graph((u, v))
+    >>> g = dgl.graph(([0,0,0,1,1,2,3,3], [1,2,3,0,3,0,0,1]))
     >>> edata = th.rand(8, 16)
+    >>> edata = th.cat(
```

Comment:
add a comment here to describe why we need zero paddings in edata (-1 paddings in `path` tensor)